packages/office-ui-fabric-react/.eslintrc.js

```diff
@@ -0,0 +1,28 @@
+// @ts-check
+const { configHelpers } = require('@fluentui/eslint-plugin');
+
+/** @type {import("eslint").Linter.Config} */
+module.exports = {
+  extends: ['plugin:@fluentui/eslint-plugin/react--legacy'],
+  root: true,
+  rules: {
+    'react/forbid-component-props': 'off',
+    '@typescript-eslint/no-explicit-any': 'off',
+    'import/no-extraneous-dependencies': [
+      'error',
+      {
+        devDependencies: [...configHelpers.devDependenciesFiles, 'src/common/{shallowUntilTarget,testUtilities}.ts'],
+      },
+    ],
+    // "import-blacklist": [true, { "../../Styling": ["FontSizes"] }]
```

Comment:
Should we remove this?

Reply:
This is one that I haven't managed to find an eslint alternative for yet, so it was left there kind of as a reminder (though not a very useful one if I forget to follow up on it)

Reply:
I realized this rule would be pretty easy to reimplement, so I wrote a version which is now added to the PR. Same with `jsx-ban-props`.